ticdc: adjust pulsar sink doc (#18171)
pingcap/docs: ticdc/ticdc-sink-to-pulsar.md

| Parameter           | Description                                                   |
| :------------------ | :------------------------------------------------------------ |

> | `pulsar`             | The scheme for the downstream Pulsar. Supported schemas are `pulsar`, `pulsar+ssl`, `pulsar+http`, and `pulsar+https`. Starting from v8.2.0, TiCDC supports `pulsar+http` and `pulsar+https`.                            |

Reviewer: ```suggestion
| `pulsar`             | The scheme for the downstream Pulsar. The value can be `pulsar`, `pulsar+ssl`, `pulsar+http`, or `pulsar+https`, in which `pulsar+http` and `pulsar+https` are supported starting from v8.2.0.                        |
```
